specification/cdn/resource-manager/Microsoft.Cdn/preview/2022-01-01-preview/cdn.json

```diff
@@ -0,0 +1,6813 @@
+      "type": "object",
+      "properties": {
+        "error": {
+          "description": "The error object.",
+          "$ref": "#/definitions/ErrorDetail"
+        }
+      }
+    },
+    "ErrorDetail": {
+      "description": "The error detail.",
+      "type": "object",
+      "properties": {
+        "code": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The error code."
+        },
+        "message": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The error message."
+        },
+        "target": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The error target."
+        },
+        "details": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ErrorDetail"
+          },
+          "description": "The error details."
+        },
+        "additionalInfo": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ErrorAdditionalInfo"
+          },
+          "description": "The error additional info."
+        }
+      }
+    },
+    "ErrorAdditionalInfo": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The additional info type."
+        },
+        "info": {
+          "readOnly": true,
+          "type": "object",
+          "description": "The additional info."
+        }
+      },
+      "description": "The resource management error additional info."
+    },
+    "Policy": {
+      "description": "A policy is a logical grouping of policies",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/PolicyProperties"
+        }
+      }
+    },
+    "PolicyProperties": {
+      "description": "The JSON object that contains the properties of the Policy to create.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/PolicyGetParameters"
+        }
+      ],
+      "required": [
+        "CustomerConfig"
+      ],
+      "properties": {
+        "CustomerConfig":{
+          "description": "Policy to be applied",
+          "type": "string"
+        }
+      }
+    },
+    "PolicyGetParameters":{
+      "description": "The JSON object that contains the properties of the Policy to update.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "./afdx.json#/definitions/AFDStateProperties"
+        }
+      ],
+      "required": [
+        "ServiceType",
+        "SchemaVersion"
+      ],
+      "SchemaVersion":{
+        "description": "Schema version of the policy",
+        "type": "string"
+      },
+      "ServiceType": {
```

Comment:
Will ServiceType support more than one value? If so can it be changed into an enum?